linetools/isgm/abscomponent.py

```diff
@@ -243,15 +243,21 @@ def from_dict(cls, idict, coord=None, **kwargs):
         else:
             radec = SkyCoord(ra=idict['RA'], dec=idict['DEC'], unit='deg')
         # Init
-        #slf = cls(radec, tuple(idict['Zion']), idict['zcomp'], Quantity(idict['vlim'], unit='km/s'),
+        # slf = cls(radec, tuple(idict['Zion']), idict['zcomp'], Quantity(idict['vlim'], unit='km/s'),
 
         # backwards compatibility
-        for key in ['reliability', 'Reliability']:
-            if key in idict.keys():
-                reliability = idict[key]
-                break
-            else:
-                reliability = 'none'
+        if 'reliability' in idict.keys():
```

Comment:
I can't see how the old code doesn't do exactly the same thing.
But you must have a reason..

Reply:
I found it clearer in the new code. but should be doing the same.